app/views/merchant_items/show.html.erb

```diff
@@ -0,0 +1,3 @@
+<h2>Name: <%= @item.name %></h2>
+<h4>Description: <%= @item.description %></h4>
```

Comment:
I really appreciate the size convention here, with the title as an h2 and the sub-information as h4.